rollup.config.ts

```diff
@@ -97,4 +98,32 @@ const buildNode = {
   ],
 }
 
-export default [buildNode, buildEsm, buildUdm]
+const buildNative = {
+  input: 'src/native/index.ts',
+  external: ['events', 'node-request-interceptor'],
+  output: {
+    file: 'native/index.js',
+    format: 'cjs',
+  },
+  plugins: [
+    alias({
+      entries: [{ find: 'timers', replacement: '../utils/reactNativeTimers' }],
```

Comment:
Let's think of it this way: for React Native the `setTimeout` behavior should remain as usual, no specifics here. It is the Node build target that requires to swap `global.setTimeout` with `timers.setTimeout` in order to be used with fake timers in Jest.

I suggest we provide a module alias for Node build target, and leave React Native configuration without custom aliases. What do you think about this?

Reply:
Seems a good idea, I made the change